### ISSUE TITLE
Noise after training for a lot of epochs: How to solve.

### ISSUE THREAD

**Issue by ysig** [archinetai/audio-diffusion-pytorch-trainer#15]
Hi, 

I train `base_medium` on a custom dataset, but I get noisy outputs even after a lot of epochs (although the noise spectrogram is not the same as the uniform one of the beginning of training).

I wanted to thus ask:

1. Should the inputs be preprocessed in a certain way?
2. How many iterations (number of samples not number epochs) are expected to have visible improvement?
3. Did you encountered collapse scenarios while training and if yes how do they look like and how did you tackle them. For example in my case just decreasing the learning rate still yielded bad results.

Thanks for open-sourcing and supporting this work!
YS
